SU-SWS/stanford_components
# EARTH-000 added some field pattern theme suggestions

# READY FOR REVIEW

# Summary
- Added extra theme suggestions for fields within pattern regions.
- Given a field `field_foo` that is in the region `items` in the pattern `section_header` this provides the following theme suggestions:
  - field--pattern.html.twig
  - field--pattern--section-header.html.twig
  - field--pattern--items.html.twig
  - field--pattern--section_header--items.html.twig

# Needed By (Date)
- When we want it.

# Urgency
- Meh, kinda if we dont wanna restyle some stuff.

# Steps to Test

1. checkout branch, `drush cr`
2. create template in theme named something like `field--pattern--items.html.twig`. i used a pretty simple template like:
```
{% for item in items %}
  {{ item.content }}
{% endfor %}
```
3. ensure the pattern is used for fields within a region with that name.

# Associated Issues and/or People
- [EARTH-16](https://stanfordits.atlassian.net/browse/EARTH-16) ![screen shot 2017-07-10 at 9 25 04 am](https://user-images.githubusercontent.com/7185045/28028543-b72ba686-6551-11e7-9425-b6921e8f77f8.png)
- Other occurances of where styles from the pattern doesn't get applied because of the extra divs that fields produces.
